TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/ProxyInterface.php

```diff
@@ -17,12 +17,4 @@
  */
 interface ProxyInterface
 {
-    /**
-     * Wake up method.
-     *
-     * Proxies need to have one as at least session handling relies on it.
-     *
-     * @return void
-     */
-    public function __wakeup();
```

Comment:
Yes actually I don't know why it was in the interface in the first place. In fact every proxy had a __wakeup method before this change so it was safe to put it in the interface. But if you see 

\TYPO3\Flow\Object\DependencyInjection\ProxyClassBuilder:147 (with the change) then I generate only $setRelatedEntitiesCode if we also generated the matching __sleep method that produces the array. If that did not happen the __wakeup method would be empty and will therefore not be rendered by our proxy builder. So now there will be classes with no autogenerated __wakeup method (because it's not needed) and therefore it must be removed from the interface otherwise -> fatal